src/test/java/uk/gov/hmcts/reform/sscs/services/ccd/UpdateCcdRepresentativeTest.java

```diff
@@ -102,21 +107,28 @@ public void givenARepContactChange_willChangeDataAndReturnTrue() {
                         .contact(Contact.builder().email("harry@potter.com").build()).build()
                 ).build())
             .subscriptions(Subscriptions.builder().representativeSubscription(
-                Subscription.builder().email("harry@potter.com").subscribeEmail(NO).subscribeSms(NO).build()).build())
+                Subscription.builder().tya(ABCDEFGH_1).email("harry@potter.com").subscribeEmail(NO)
+                        .subscribeSms(NO).build()).build())
             .build();
 
-        SscsCaseData existingCaseData = SscsCaseData.builder().appeal(Appeal.builder()
-            .rep(
-                Representative.builder().name(Name.builder().lastName("Potter").build())
-                    .contact(Contact.builder().email("harry.potter@wizards.com").build()).build()
-            ).build())
+        SscsCaseData existingCaseData = SscsCaseData.builder()
+            .appeal(Appeal.builder()
+                .rep(
+                    Representative.builder().name(Name.builder().lastName("Potter").build())
+                        .contact(Contact.builder().email("harry.potter@wizards.com").build()).build()
+                ).build())
+            .subscriptions(Subscriptions.builder().representativeSubscription(
+                Subscription.builder().tya(ABCDEFGH_2).email("harry.potter@wizards.com").subscribeEmail(NO)
+                        .subscribeSms(NO).build()).build())
             .build();
 
         boolean hasDataChanged = UpdateCcdRepresentative.updateCcdRepresentative(gapsCaseData, existingCaseData);
         assertTrue("rep contact has changed", hasDataChanged);
         assertEquals(gapsCaseData.getAppeal().getRep(), existingCaseData.getAppeal().getRep());
-        assertEquals(gapsCaseData.getSubscriptions().getRepresentativeSubscription(),
-            existingCaseData.getSubscriptions().getRepresentativeSubscription());
+        // should not change the existing tya number even though we have new tya number on gapsCaseData
```

Comment:
could you remove the comment and move it into a method with a meaningful name, please.